<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="5" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Surefire suite" duration-ms="9090" started-at="2016-04-06T16:40:57Z" finished-at="2016-04-06T16:41:07Z">
    <groups>
    </groups>
    <test name="Surefire test" duration-ms="9090" started-at="2016-04-06T16:40:57Z" finished-at="2016-04-06T16:41:07Z">
      <class name="com.masteringselenium.BasicTestWD">
        <test-method status="PASS" signature="instantiateDriverObject()[pri:0, instance:com.masteringselenium.BasicTestWD@21bcffb5]" name="instantiateDriverObject" is-config="true" duration-ms="6" started-at="2016-04-06T11:40:57Z" finished-at="2016-04-06T11:40:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- instantiateDriverObject -->
        <test-method status="PASS" signature="googleDohniExample()[pri:0, instance:com.masteringselenium.BasicTestWD@21bcffb5]" name="googleDohniExample" duration-ms="9071" started-at="2016-04-06T11:40:57Z" finished-at="2016-04-06T11:41:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- googleDohniExample -->
        <test-method status="PASS" signature="googleGangulyExample()[pri:0, instance:com.masteringselenium.BasicTestWD@21bcffb5]" name="googleGangulyExample" duration-ms="7455" started-at="2016-04-06T11:40:57Z" finished-at="2016-04-06T11:41:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- googleGangulyExample -->
        <test-method status="FAIL" signature="googleCheeseExample()[pri:0, instance:com.masteringselenium.BasicTestWD@21bcffb5]" name="googleCheeseExample" duration-ms="3985" started-at="2016-04-06T11:40:57Z" finished-at="2016-04-06T11:41:01Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertFalse(Assert.java:63)
	at org.testng.Assert.assertFalse(Assert.java:73)
	at com.masteringselenium.BasicTestWD.googleCheeseExample(BasicTestWD.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- googleCheeseExample -->
        <test-method status="PASS" signature="quitDriver()[pri:0, instance:com.masteringselenium.BasicTestWD@21bcffb5]" name="quitDriver" is-config="true" duration-ms="8" started-at="2016-04-06T11:41:02Z" finished-at="2016-04-06T11:41:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method status="PASS" signature="googleMilkExample()[pri:0, instance:com.masteringselenium.BasicTestWD@21bcffb5]" name="googleMilkExample" duration-ms="1595" started-at="2016-04-06T11:41:02Z" finished-at="2016-04-06T11:41:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- googleMilkExample -->
        <test-method status="PASS" signature="quitDriver()[pri:0, instance:com.masteringselenium.BasicTestWD@21bcffb5]" name="quitDriver" is-config="true" duration-ms="10" started-at="2016-04-06T11:41:03Z" finished-at="2016-04-06T11:41:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method status="FAIL" signature="googleSachineExample()[pri:0, instance:com.masteringselenium.BasicTestWD@21bcffb5]" name="googleSachineExample" duration-ms="1783" started-at="2016-04-06T11:41:03Z" finished-at="2016-04-06T11:41:05Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at com.masteringselenium.BasicTestWD.googleSachineExample(BasicTestWD.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- googleSachineExample -->
        <test-method status="PASS" signature="quitDriver()[pri:0, instance:com.masteringselenium.BasicTestWD@21bcffb5]" name="quitDriver" is-config="true" duration-ms="8" started-at="2016-04-06T11:41:05Z" finished-at="2016-04-06T11:41:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method status="PASS" signature="quitDriver()[pri:0, instance:com.masteringselenium.BasicTestWD@21bcffb5]" name="quitDriver" is-config="true" duration-ms="7" started-at="2016-04-06T11:41:05Z" finished-at="2016-04-06T11:41:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method status="PASS" signature="quitDriver()[pri:0, instance:com.masteringselenium.BasicTestWD@21bcffb5]" name="quitDriver" is-config="true" duration-ms="7" started-at="2016-04-06T11:41:07Z" finished-at="2016-04-06T11:41:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver -->
        <test-method status="PASS" signature="closeDriverObjects()[pri:0, instance:com.masteringselenium.BasicTestWD@21bcffb5]" name="closeDriverObjects" is-config="true" duration-ms="262" started-at="2016-04-06T11:41:07Z" finished-at="2016-04-06T11:41:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDriverObjects -->
      </class> <!-- com.masteringselenium.BasicTestWD -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
